index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="header">
+    <div class="up"><img src="icons/Dark.png"></div>
+    <div class="logo">My Logo</div>
+    <div class="union"><img src="icons/Union.png"></div>
+  </div> 
+  <div class="headline">
+    <div class="text">Hey There. <br> I'm Alex.</div>
+    <div class="text2">I'm a Software Developer</div>
+    <div class="text3">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</div>
```

Comment:
Kindly update this element to use a <p> tag. This is for semantic reasons, as this element represents a paragraph🙏